src/game_server/command.rs

```diff
@@ -41,6 +42,39 @@ impl SerializePacket for CommandOpCode {
     }
 }
 
+#[derive(SerializePacket, DeserializePacket)]
+pub struct Rgba {
+    r: u8,
+    g: u8,
+    b: u8,
+    a: u8,
+}
+
+impl Rgba {
+    pub fn new(b: u8, g: u8, r: u8, a: u8) -> Self {
```

Comment:
We would need to keep the previous ordering of the fields inside the struct for it to serialize correctly, right? But we can change the params to this `new` method to RGBA order without affecting the (de)serialization